features/itunes/components/TrackCard/index.tsx

```diff
@@ -1,16 +1,18 @@
-import { isEmpty } from "lodash";
+import { useRouter } from "next/router";
 import styled from "styled-components";
+import { isEmpty } from "lodash";
```

Comment:
Generally when using lodash in frontend it is recommended to destructure it saves a lot of initial load time and space.

```suggestion
import isEmpty from "lodash/isEmpty";
```

Reply:
Oh, thanks for the update. Will update it accordingly

Reply:
Updated accordingly in the latest commit.